 # output from make_prediction.sh
Port: 8000
{
  "prediction": [
    20.35373177134412
  ]
}
 #output from run_kubernetes window

Error from server (AlreadyExists): pods "udacityprojectfive" already exists
NAME                 READY   STATUS    RESTARTS   AGE
udacityprojectfive   1/1     Running   0          37m
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000

(.devops) maria@maria:~/Desktop/cloned/DevOps_Microservices/project-ml-microservice-kubernetes$ kubectl logs udacityprojectfive
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 129-516-115
[2020-09-05 08:51:18,616] INFO in app: JSON payload: 
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2020-09-05 08:51:18,641] INFO in app: Inference payload DataFrame: 
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-09-05 08:51:18,651] INFO in app: Scaling Payload: 
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-09-05 08:51:18,655] INFO in app: Prediction value: [20.35373177134412]
127.0.0.1 - - [05/Sep/2020 08:51:18] "POST /predict HTTP/1.1" 200 -
